File: frontend/src/app/dto/learning-schedule/learning-schedule.data.dto.ts
export class LearningScheduleDataDto{

    private _id! : number
	private _startTime! : string
	private _endTime! : string
	private _learningId! : number
	private _learningCode! : string
	private _learningSubject! : string
	private _learningDescription! : string
	private _ver! : number
	private _isActive! : boolean

    public get id(): number {
        return this._id;
    }

    public set id(id: number) {
        this._id = id;
    }

    public get startTime(): string {
        return this._startTime;
    }

    public set startTime(startTime: string) {
        this._startTime = startTime;
    }

    public get endTime(): string {
        return this._endTime;
    }

    public set endTime(endTime: string) {
        this._endTime = endTime;
    }

    public get learningId(): number {
        return this._learningId;
    }

    public set learningId(learningId: number) {
        this._learningId = learningId;
    }

    public get learningCode(): string {
        return this._learningCode;
    }

    public set learningCode(learningCode: string) {
        this._learningCode = learningCode;
    }

    public get learningSubject(): string {
        return this._learningSubject;
    }

    public set learningSubject(learningSubject: string) {
        this._learningSubject = learningSubject;
    }

    public get learningDescription(): string {
        return this._learningDescription;
    }

    public set learningDescription(learningDescription: string) {
        this._learningDescription = learningDescription;
    }

    public get ver(): number {
        return this._ver;
    }

    public set ver(ver: number) {
        this._ver = ver;
    }

    public get isActive(): boolean {
        return this._isActive;
    }

    public set isActive(isActive: boolean) {
        this._isActive = isActive;
    }
   
}

File: backend/src/main/java/com/lawencon/elearn/dto/learningschedule/LearningScheduleDataDto.java
package com.lawencon.elearn.dto.learningschedule;

import java.time.LocalDateTime;

public class LearningScheduleDataDto {
	private Long id;
	private LocalDateTime startTime;
	private LocalDateTime endTime;
	private Long learningId;
	private String learningCode;
	private String learningSubject;
	private String learningDescription;
	private Integer ver;
	private Boolean isActive;
	public Boolean getIsActive() {
		return isActive;
	}
	public void setIsActive(Boolean isActive) {
		this.isActive = isActive;
	}
	public Long getId() {
		return id;
	}
	public void setId(Long id) {
		this.id = id;
	}
	public LocalDateTime getStartTime() {
		return startTime;
	}
	public void setStartTime(LocalDateTime startTime) {
		this.startTime = startTime;
	}
	public LocalDateTime getEndTime() {
		return endTime;
	}
	public void setEndTime(LocalDateTime endTime) {
		this.endTime = endTime;
	}
	public Long getLearningId() {
		return learningId;
	}
	public void setLearningId(Long learningId) {
		this.learningId = learningId;
	}
	public String getLearningCode() {
		return learningCode;
	}
	public void setLearningCode(String learningCode) {
		this.learningCode = learningCode;
	}
	public String getLearningSubject() {
		return learningSubject;
	}
	public void setLearningSubject(String learningSubject) {
		this.learningSubject = learningSubject;
	}
	public String getLearningDescription() {
		return learningDescription;
	}
	public void setLearningDescription(String learningDescription) {
		this.learningDescription = learningDescription;
	}
	public Integer getVer() {
		return ver;
	}
	public void setVer(Integer ver) {
		this.ver = ver;
	}
}
